<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1058</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Konfiguracja projektu</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="minimumSize">
                <size>
                 <width>141</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Paczki</string>
               </property>
               <property name="buddy">
                <cstring>packs</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="newPack">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Nowa</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="removePack">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Usuń</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QListWidget" name="packs"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Folder wyjściowy</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="outputFolder">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="outputFolderBtn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="packSettings">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Ustawienia paczki</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Nazwa paczki</string>
             </property>
             <property name="buddy">
              <cstring>activePackName</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="activePackName">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Format wyjściowy</string>
             </property>
             <property name="buddy">
              <cstring>outPutFormat</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="outPutFormat">
             <item>
              <property name="text">
               <string>PNG</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Szerokość</string>
             </property>
             <property name="buddy">
              <cstring>widthPack</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="widthPack">
             <property name="suffix">
              <string>px</string>
             </property>
             <property name="maximum">
              <number>9999</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
             <property name="value">
              <number>1024</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Wysokość</string>
             </property>
             <property name="buddy">
              <cstring>heightPack</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="heightPack">
             <property name="suffix">
              <string>px</string>
             </property>
             <property name="maximum">
              <number>9999</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
             <property name="value">
              <number>1024</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="inputFolder">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Folder wejściowy</string>
             </property>
             <property name="buddy">
              <cstring>inputFolderDir</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="inputFolderDir"/>
           </item>
           <item>
            <widget class="QPushButton" name="inputFolderBtn">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QPushButton" name="packAtlas">
             <property name="text">
              <string>Pakuj</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="saveSettings">
             <property name="text">
              <string>Zapisz ustawienia</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="preview">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="styleSheet">
         <string notr="true">background-color: transparent;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1058</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionZapisz"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuPomoc">
    <property name="title">
     <string>Pomoc</string>
    </property>
    <addaction name="about"/>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuPomoc"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Wyjście</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>O programie</string>
   </property>
  </action>
  <action name="actionZapisz">
   <property name="text">
    <string>Zapisz</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
